openebs/form.py

```diff
@@ -12,13 +12,24 @@
 from openebs.models import Kv15Stopmessage, Kv15Scenario, Kv15ScenarioMessage, Kv17Change, get_end_service
 from openebs.models import Kv17JourneyChange
 from utils.time import get_operator_date
+from dateutil import parser
+from datetime import datetime
 
 log = logging.getLogger('openebs.forms')
 
 
 class Kv15StopMessageForm(forms.ModelForm):
     def clean(self):
         # TODO Move _all_ halte parsing here!
+
+        endtime = parser.parse(self.data['messageendtime'])
+        if not is_aware(endtime):
```

Comment:
There must be a better way of doing this

Reply:
Do you mean this one:
    from django.utils.dateparse import parse_date  ?